fork-off/Cargo.toml

```diff
@@ -1,18 +1,16 @@
 [package]
 name = "fork-off"
-version = "1.0.1"
+version = "1.2.0"
 edition = "2021"
 
 [dependencies]
-# client
-substrate-api-client = { git = "https://github.com/Cardinal-Cryptography/substrate-api-client.git", branch = "polkadot-v0.9.19" }
```

Comment:
While I agree that dependency on `substrate-api-client` is not ideal, but I believe we can w/a this by depending on some core frames from Substrate to access common storage-releated code, see other comments in this PR

Reply:
not relevant I believe